x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/header_actions/header_actions.test.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { apiIndex, connectorIndex, crawlerIndex } from '../../../../__mocks__/view_index.mock';
+
+import React from 'react';
+
+import { CrawlerStatusIndicator } from '../../../shared/crawler_status_indicator/crawler_status_indicator';
+
+import { getHeaderActions } from './header_actions';
+import { SearchEnginesPopover } from './search_engines_popover';
+import { SyncsContextMenu } from './syncs_context_menu';
+
+describe('Header Actions', () => {
+  it('renders api index', () => {
+    expect(getHeaderActions(apiIndex)).toEqual([
+      <SearchEnginesPopover indexName="api" ingestionMethod="api" isHiddenIndex={false} />,
+    ]);
+  });
+  it('renders connector index', () => {
+    expect(getHeaderActions(connectorIndex)).toEqual([
+      <SyncsContextMenu />,
```

Comment:
non-blocking question: why do we compare to JSX  vs using snapshots?

Reply:
Honestly: 'cause it's not in my workflow to take snapshots. Probably good to take them going forward! I'll put it on the frontend sync agenda.